### PR TITLE
[fix] fixed TS.MGET reply format. included WITHLABELS option support

### LINKED ISSUE #315: inconsistencies across returned value on time-series between TS.MGET and TS.MRANGE
Nothwistanding the `WITHLABELS` behaviour, please denote positions 3 and 4 of the reply of mget, and position 3 of the reply of mrange. We are returning the point of the time-serie in `MGET` in the incorrect manner which breaks the clients and creates inconsistencies. Position 3 of the `TS.MGET` reply should an array, which will also contain an array of 1 point to be consistent with mrange reply.  
```
"TS.ADD" "test_TestClient_MultiGet_key2" "1" "5" "LABELS" "metric" "cpu" "country" "UK"
```
# inconsistent reply format on `TS.MGET`
```
redis-cli -a SUPERSECRET "TS.MGET" "FILTER" "metric=cpu" "country=UK"
Warning: Using a password with '-a' or '-u' option on the command line interface may not be safe.
1) 1) "test_TestClient_MultiGet_key2"
   2) 1) 1) "metric"
         2) "cpu"
      2) 1) "country"
         2) "UK"
   3) (integer) 1
   4) "5"
```
# correct reply format on `TS.MRANGE`
```
redis-cli -a SUPERSECRET "TS.MRANGE" - + "FILTER" "metric=cpu" "country=UK"
Warning: Using a password with '-a' or '-u' option on the command line interface may not be safe.
1) 1) "test_TestClient_MultiGet_key2"
   2) (empty array)
   3) 1) 1) (integer) 1
         2) "5"
```

### LINKED ISSUE #316: support for optional arg `WITHLABELS` on `TS.MGET` command
to be consistent with `MRANGE` 
this does not break any client since the change is to reply an empty position

### PR DESCRIPTION
**Pull request by filipecosta90**
fixes #315 
fixes #316 

- [fix] fixed TS.MGET reply format to match the one from other multi timeseries replies
- [add] include WITHLABELS optional argument. 
- [add] updated documentation and included complexity of command

### CODE REVIEW

File: docs/commands.md [hunk 1]
```diff
@@ -379,28 +379,62 @@ TS.GET key
...
 ```sql
 127.0.0.1:6379> TS.MGET FILTER area_id=32
 1) 1) "temperature:2:32"
    2) 1) 1) "sensor_id"
          2) "2"
       2) 1) "area_id"
          2) "32"
-   3) (integer) 1548149181000
-   4) "30"
+      3) 1) 1) (integer) 1548149181000
```
Comment (ashtul) [inline docs/commands.md]: This should be one tab back. Same with next one

File: docs/commands.md [hunk 2]
```diff
@@ -379,28 +379,62 @@ TS.GET key
...
+n = Number of time-series that match the filters
+
+#### Examples
+
+##### MGET Example with default behaviour
+```sql
+127.0.0.1:6379> TS.MGET FILTER area_id=32
+1) 1) "temperature:2:32"
+   2) (empty list or set)
+      3) 1) 1) (integer) 1548149181000
```
Comment (ashtul) [inline docs/commands.md]: Fix tabs

File: src/module.c [hunk 3]
```diff
@@ -304,6 +305,17 @@ void ReplyWithSeriesLabels(RedisModuleCtx *ctx, const Series *series) {
     }
 }
 
+void ReplyWithSeriesLastDatapoint(RedisModuleCtx *ctx, const Series *series) {
+    if(SeriesGetNumSamples(series)==0){
+        RedisModule_ReplyWithArray(ctx, 0);
```
Comment (ashtul) [inline src/module.c]: Please cover in test

Comment (filipecosta90) [reply src/module.c]: @ashtul please revise latest commit with extra testing on `mget` with series with empty data.
